docs/concepts/cluster-administration/networking.md

```diff
@@ -190,6 +190,12 @@ Lars Kellogg-Stedman.
 
 Multus supports all [reference plugins](https://github.com/containernetworking/plugins) (eg. [Flannel](https://github.com/containernetworking/plugins/tree/master/plugins/meta/flannel), [DHCP](https://github.com/containernetworking/plugins/tree/master/plugins/ipam/dhcp), [Macvlan](https://github.com/containernetworking/plugins/tree/master/plugins/main/macvlan)) that implement the CNI specification and 3rd party plugins (eg. [Calico](https://github.com/projectcalico/cni-plugin), [Weave](https://github.com/weaveworks/weave), [Cilium](https://github.com/cilium/cilium), [Contiv](https://github.com/contiv/netplugin)). In addition to it, Multus supports [SRIOV](https://github.com/hustcat/sriov-cni), [DPDK](https://github.com/Intel-Corp/sriov-cni), [OVS-DPDK & VPP](https://github.com/intel/vhost-user-net-plugin) workloads in Kubernetes with both cloud native and NFV based applications in Kubernetes.
 
+### NSX-T
+ 
+[VMware NSX-T] (https://docs.vmware.com/en/VMware-NSX-T/index.html) is a network virtualization and security platform. NSX-T can provide network virtualization for a multi-cloud and multi-hypervisor environment and is focused on emerging application frameworks and architectures that have heterogeneous endpoints and technology stacks. In addition to vSphere hypervisors, these environments include other hypervisors such as KVM, containers, and bare metal.
+ 
+[NSX-T Container Plug-in (NCP)] (https://docs.vmware.com/en/VMware-NSX-T/2.0/nsxt_20_ncp_kubernetes.pdf) provides integration between NSX-T and container orchestrators such as Kubernetes, as well as integration between NSX-T and container-based CaaS / PaaS platforms such as Pivotal Container Service (PKS) and Openshift. 
+
```

Comment:
You can see in the [preview](https://deploy-preview-5987--kubernetes-io-master-staging.netlify.com/docs/concepts/cluster-administration/networking/#nsx-t) that these links still aren't working. I think the problem is that you have a space between [...] and (...).

[Working example](https://docs.vmware.com/en/VMware-NSX-T/index.html)

[Not working example] (https://docs.vmware.com/en/VMware-NSX-T/index.html)

Reply:
Corrected. I hope now it is good